sabnzbd/downloader.py

```diff
@@ -654,13 +657,13 @@ def run(self):
                         # In case nzf has disappeared because the file was deleted before the update could happen
                         pass
 
+                    sabnzbd.BPSMeter.update(server.id, bytes_received, force_full_update=False)
```

Comment:
Why move it up? This way it will still display 20.1MB/s for example even when user has set a limit of 20MB/s.
I know it's a bit fake, but has always worked like that.

Reply:
Mostly to make sure the amount is included for the next test, but if you think that gives the wrong number then it's probably ok to move it below.

Reply:
I just tested with 7 000 000 bytes as limit and a 50% speed limit. This results in a speed of 3492500 - 3495500 B/s when I debug log the bps as bytes from the BPS meter. It doesn't matter if the update call is above or below. Is it still too high if you measure it in bytes?